paasta_tools/cli/cmds/status.py

```diff
@@ -74,6 +82,12 @@ def add_subparser(subparsers):
         default=DEFAULT_SOA_DIR,
         help="define a different soa config directory",
     )
+    status_parser.add_argument(
```

Comment:
I don't feel too strongly about this, but a command line switch seems the wrong interface for this. how about a ``USE_PAASTA_API`` environment variable?